osidb/tests/test_endpoints.py

```diff
@@ -1951,6 +1953,265 @@ def test_tracker_delete(self, auth_client, test_api_uri):
         # with the authoritative sources of the tracker data
         assert response.status_code == 405
 
+    def test_packageversions_filter(self, auth_client, embargo_access, test_api_uri):
+        """
+        Test the non-trivial parts of FlawPackageVersionFilter via REST API GET requests.
+        """
+        # Source must be private in order for validation to pass.
+        flaw = FlawFactory(source=FlawSource.CUSTOMER)
```

Comment:
This should be written as `flaw = FlawFactory()` (more occurrences).

Having a flaw with a public source (i.e. where the flaw was first reported) is a valid option. `package_versions` does not require any restriction.

Reply:
Thanks for spotting this. Resolved in commit 53f1e57a1461da2cc2a2cbdeceb114ad1389d482